Directory.Build.props

```diff
@@ -19,6 +19,7 @@
          For clarity, without PrivateAssets marked here, anyone consuming Microsoft.Identity.Client
          would also be forced to install these dependencies.  PrivateAssets avoids this problem. -->
     <PackageReference Include="AsyncUsageAnalyzers" Version="1.0.0-alpha003" PrivateAssets="All" />
-    <PackageReference Include="Microsoft.CodeAnalysis.FxCopAnalyzers" Version="2.6.3" PrivateAssets="All" />    
+    <PackageReference Include="Microsoft.CodeAnalysis.FxCopAnalyzers" Version="2.6.3" PrivateAssets="All" />
```

Comment:
This analyzer has been deprecated, see https://docs.microsoft.com/en-us/visualstudio/code-quality/migrate-from-fxcop-analyzers-to-net-analyzers?view=vs-2019
It has been replaced with Microsoft.CodeAnalysis.NetAnalyzers, which you just referneced. Do we need to keep the old analyzer? 

IMO, the old ruleset is useless and should be fully replaced by the work you are doing. Keep nothing, except maybe the async analyzer (the one that warns if you don't use ConfigureAwait).

Reply:
i.e. <CodeAnalysisRuleSet>$(MSBuildThisFileDirectory)/build/SolutionWideAnalyzerConfig.ruleset</CodeAnalysisRuleSet>  can go.

Reply:
If I remember correctly, the async analyzer is also obsolete, new Roslyn analyzers have that rule (https://docs.microsoft.com/en-us/dotnet/fundamentals/code-analysis/quality-rules/ca2007?view=vs-2019)

Reply:
This is one of the errors @pmaytak but there is another one that we need from the async analyzer that I cant find. seems like they rules may have been moved to a package called Microsoft.VisualStudio.Threading but I cant find too much info on it yet